CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ ycm_default_dirs(${VARS_PREFIX})
 option(CODYCO_USES_URDFDOM "Compile yarpWholeBodyInterface with URDF support" TRUE)
 option(COMPILE_AS_SHARED_LIBRARY "Compile ${PROJECT_NAME} as a shared library" TRUE)
 
-option(ENABLE_TESTS "Enable unit testing" FALSE)
+option(YARPWHOLEBODYINTERFACE_ENABLE_TESTS "Enable unit testing" FALSE)
```

Comment:
Fixed travis file in https://github.com/robotology-playground/yarp-wholebodyinterface/commit/48999389ad886e61e8ce0417754e1f23ad31ace1.